products/redhat-openjdk.md

```diff
@@ -0,0 +1,126 @@
+    - purl: pkg:redhat/java-11-openjdk-devel-debug
+
+releases:
+-   releaseCycle: "17"
+    releaseDate: 2021-05-18 # RHEL 8.4
+    eol: 2027-10-30
+    # https://access.redhat.com/downloads/content/java-17-openjdk/x86_64/package-latest
+    latest: "17.0.6.0.10-6"
+    latestReleaseDate: 2023-04-26
+    link: https://access.redhat.com/documentation/en-us/openjdk/17/html-single/release_notes_for_openjdk_17.0.6/index
+
+-   releaseCycle: "11"
+    releaseDate: 2018-10-30 # RHEL 7.6
+    eol: 2024-10-30
+    # https://access.redhat.com/downloads/content/java-11-openjdk/x86_64/package-latest
+    latest: "11.0.19.0.7-2"
+    latestReleaseDate: 2023-04-26
+
+# 8 is a tricky release since its not supported on RHEL6
+# So we mark it twice, once specifically for RHEL6
+-   releaseCycle: "8"
+    releaseLabel: "OpenJDK 8 (1.8) - Windows / RHEL 7+"
+    releaseDate: 2015-03-05 # RHEL 7.1
+    eol: 2026-11-30
+    # https://access.redhat.com/downloads/content/java-1.8.0-openjdk/x86_64/package-latest
+    latest: "1.8.0.372.b07-3"
+    latestReleaseDate: 2023-04-27
+
+-   releaseCycle: "rhel6-8"
+    releaseLabel: "OpenJDK 8 (1.8) - RHEL 6"
+    releaseDate: 2014-10-14 # RHEL 6.6
+    eol: 2020-11-30
+    # The last release RHEL6 was 1.8.0_275 as per the docs
+    # filtered to find the latest in the series as per
+    # https://access.redhat.com/downloads/content/java-1.8.0-openjdk/x86_64/package-latest
+    latest: "1.8.0.275.b01-1"
+    latestReleaseDate: 2020-11-06
+
+-   releaseCycle: "7"
+    releaseLabel: "OpenJDK 7 (1.7)"
+    releaseDate: 2013-01-07 # RHEL 5.9
+    eol: 2020-06-30
+    # https://access.redhat.com/downloads/content/java-1.7.0-openjdk/x86_64/package-latest
+    latest: "1.7.0.261-2.6.22.2"
+    latestReleaseDate: 2020-04-15
+
+-   releaseCycle: "6"
+    releaseLabel: "OpenJDK 6 (1.6)"
+    releaseDate: 2009-01-20 # RHEL 5.3
+    eol: 2016-12-31
+    # https://access.redhat.com/downloads/content/java-1.6.0-openjdk/x86_64/package-latest
+    latest: "1.6.0.41-1.13.13.1"
+    latestReleaseDate: 2017-01-04
+
+---
+
+> [Red Hat OpenJDK](https://access.redhat.com/products/openjdk/) is a build of the Open Java Development Kit (OpenJDK)
+> with long-term support and patches from Red Hat. It is distributed as part of Red Hat Enterprise Linux,
+> but is also available with [OpenShift](https://www.redhat.com/en/technologies/cloud-computing/openshift)
+> and [Red Hat Middleware](https://developers.redhat.com/middleware).
+
+This page covers the Red Hat OpenJDK production packages, which require a
+[valid entitlement](https://access.redhat.com/articles/1299013#OpenJDK_Entitlements).
+[Developer Builds](https://developers.redhat.com/products/openjdk/download) are released
+for personal use under the [Red Hat Developer Subscription](https://developers.redhat.com/terms-and-conditions)
+and are unsupported.
+
+Red Hat documents [the differences between Red Hat OpenJDK and Oracle JDK](https://access.redhat.com/solutions/2489791).
+
+## Support
```

Comment:
It could be nice to clearly say that Red Hat OpenJDK builds only target LTS releases of OpenJDK. Do you know how fast new LTS versions are available ?